spec.bs

```diff
@@ -3608,6 +3609,28 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
 
 </div>
 
+# Feature Detection # {#feature-detection}

```

Comment:
Suggest documenting this in https://github.com/WICG/turtledove/blob/main/PA_Feature_Detecting.md.

Reply:
Thanks, wasn't aware of this. I think I'll probably add it to the explainer CL, fix the I2S link TBD given it has a bunch of process stuff.